README.md

```diff
@@ -107,3 +107,31 @@ If you have write access to this repo, you can ship a prerelease or full release
 or
 
 `scripts/bump_version.sh`
+
+
+## Running preflight tests
+
+Find your auth token inside `~/.fly/config.yml`
+
+Create a envfile `.preflight-env` with the following content and replace the relevant values there:
+
+```
+export FLY_PREFLIGHT_TEST_FLY_ORG="flyctl-test-YOURNAME"
+export FLY_PREFLIGHT_TEST_FLY_REGIONS="region1 region2"
+export FLY_PREFLIGHT_TEST_ACCESS_TOKEN="Your super token goes here"
+export FLY_PREFLIGHT_TEST_FLY_FLYCTL_BINARY_PATH="../bin/flyctl"
+```
+
+Some recommendations:
+* Do not use your "personal" org, create an new org (i.e. `flyctl-tests-YOURNAME`)
+  Ask for 100% comp at https://flyio.discourse.team/t/employee-accounts-how-to-prevent-charges/526/95
```

Comment:
Might want to preface this with "For employees, " for clarity

Reply:
good point, fixed b933a9ff